awsagentprovider/src/test/java/software/amazon/opentelemetry/javaagent/providers/AwsTracerConfigurerTest.java

```diff
@@ -19,12 +19,20 @@
 
 import com.google.common.primitives.Ints;
 import io.opentelemetry.api.trace.TracerProvider;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.sdk.trace.SdkTracerProviderBuilder;
 import java.util.concurrent.TimeUnit;
 import org.junit.jupiter.api.RepeatedTest;
 
-class AwsTracerProviderFactoryTest {
+class AwsTracerConfigurerTest {
 
-  private static final TracerProvider tracerProvider = new AwsTracerProviderFactory().create();
+  private static final TracerProvider tracerProvider;
+
+  static {
+    SdkTracerProviderBuilder builder = SdkTracerProvider.builder();
+    new AwsTracerConfigurer().configure(builder);
```

Comment:
Why does this feel like it should be static?

Reply:
It's loaded by SPI so has to be an interface implementation, looks a bit weird in the test because of that though.